src/common/framework/params_builder.ts

```diff
@@ -292,54 +370,6 @@ export class SubcaseParamsBuilder<CaseP extends {}, SubcaseP extends {}>
   }
 }
 
-/** Creates a generator function for expandWithParams() methods above. */
```

Comment:
All this logic got moved up inline, because it was starting to get really messy (and this caused a bug that I had a really hard time debugging because the types were screwy because function(x) can be implicitly cast to function(x,y)).
Being able to specialize this above also saves a bunch of unnecessary `mergeParams({}, x)`.